ext/standard/math.c

```diff
@@ -1296,6 +1296,24 @@ PHP_FUNCTION(fmod)
 }
 /* }}} */
 
+/* {{{ proto float fdiv(float x, float y)
+   Perform floating-point division of x / y with IEEE-754 semantics for division by zero. */
+#ifdef __clang__
+__attribute__((no_sanitize("float-divide-by-zero")))
+#endif
+PHP_FUNCTION(fdiv)
```

Comment:
I recognize the etymology, but I wonder if `floatdiv` would be a more natural PHP name: it's verbose, and pairs with `intdiv` obviously. Said another way, the behavior seems closer to `intdiv` than `fmod`, so the name should be more closely aligned with the former (`floatdiv`) than the latter (`fdiv`).

Reply:
> Said another way, the behavior seems closer to intdiv than fmod, so the name should be more closely aligned with the former (floatdiv) than the latter (fdiv).

That true, but if someone would ask me a question "What is a complementary (?) function to `fmod`?" my first guess would be `fdiv`. _Just sharing my thoughs 😉_